libp2p/Host.cpp

```diff
@@ -152,21 +152,60 @@ void Host::stop()
         stopWorking();
 }
 
+void Host::startCapabilities()
+{
+    for (auto const& itCap : m_capabilities)
+    {
+        auto timer = itCap.second.backgroundWorkTimer;
+        auto cap = itCap.second.capability;
+        m_ioService.post([this, timer, cap]() {
```

Comment:
Is this done to execute the first iteration without delay?
I think just calling `scheduleCapabilityWorkLoop` for each capability would be simpler and the delay wouldn't make any difference in practice.

Reply:
Yes, you are correct. I'll remove this.